server/src/test/java/org/elasticsearch/cluster/coordination/CoordinatorTests.java

```diff
@@ -379,7 +297,7 @@ public void run() {
 
             @Override
             public String toString() {
-                return runnable.toString() + " (wrapped for " + node + ")";
+                return node.getId() + ": " + runnable.toString();
```

Comment:
I think that having the UUID of the disco node will be useful here when we start rebooting nodes.

Reply:
I've added the ephemeral id